boards/opentitan/earlgrey-cw310/run.sh

```diff
@@ -19,14 +19,22 @@ if [[ "${VERILATOR}" == "yes" ]]; then
 		--binary --offset 0 --byte-swap 8 --fill 0xff \
 		-within "$BUILD_DIR"/earlgrey-cw310-tests.bin\
 		-binary -range-pad 8 --output "$BUILD_DIR"/binary.64.vmem --vmem 64
-	${OPENTITAN_TREE}/build/lowrisc_dv_chip_verilator_sim_0.1/sim-verilator/Vchip_sim_tb \
-		--meminit=rom,${OPENTITAN_TREE}/build-out/sw/device/lib/testing/test_rom/test_rom_sim_verilator.scr.39.vmem \
+	${OPENTITAN_TREE}/bazel-out/k8-fastbuild/bin/hw/build.verilator_real/sim-verilator/Vchip_sim_tb \
+		--meminit=rom,${OPENTITAN_TREE}/bazel-out/k8-fastbuild-ST-97f470ee3b14/bin/sw/device/lib/testing/test_rom/test_rom_sim_verilator.scr.39.vmem \
 		--meminit=flash,./"$BUILD_DIR"/binary.64.vmem \
-		--meminit=otp,${OPENTITAN_TREE}/build-out/sw/device/otp_img/otp_img_sim_verilator.vmem
+		--meminit=otp,${OPENTITAN_TREE}/bazel-out/k8-fastbuild/bin/sw/device/tests/otp_ctrl_smoketest_sim_verilator.runfiles/lowrisc_opentitan/hw/ip/otp_ctrl/data/img_rma.vmem
 elif [[ "${OPENTITAN_TREE}" != "" ]]; then
 	riscv64-linux-gnu-objcopy --update-section .apps=${APP} ${1} bundle.elf
 	riscv64-linux-gnu-objcopy --output-target=binary bundle.elf binary
-	${OPENTITAN_TREE}/util/fpga/cw310_loader.py --firmware binary
+
+	${OPENTITAN_TREE}/build-out/sw/host/rom_ext_signer \
```

Comment:
This utility (rom_ext_signer) is gone.  It's functionality has been subsumed into opentitantool.

Signing the image here is not required and somewhat academic.  The test_rom does not care about signatures.  The mask_rom does, however, in the long term, the mask ROM will not directly boot the OS.  The mask ROM will boot a block of firmware called the ROM_EXT which will have further low-level responsibilities in initializing the chip before booting owner code.

While it is an interesting exercise to see the mask ROM validate a signature, signing at this time (and with the key below) is premature.